format/private/format.sh

```diff
@@ -54,15 +54,15 @@ function ls-files {
         exit 1
         ;;
     esac
-    
+
     if [ "$#" -eq 0 ]; then
         # When the formatter is run with no arguments, we run over "all files in the repo".
         # However, we want to ignore anything that is in .gitignore, is marked for delete, etc.
         # So we use `git ls-files` with some additional care.
 
         # TODO: determine which staged changes we should format; avoid formatting unstaged changes
         # TODO: try to format only modified regions of the file (where supported)
-        git ls-files --cached --modified --other --exclude-standard ${patterns[@]} | {
+        git ls-files --cached --modified --other --exclude-standard "${patterns[@]}" | {
```

Comment:
@IamXander adding in a ` | tee /dev/stderr ` here is helpful for debugging, doing the same in #124 will show that even there only the root BUILD file is being considered without the add'l pattern expansion